### ISSUE TITLE
Strange number after theme "fractal summer colors" author for "Italiano"

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla/addons-frontend#2191]
Steps to reproduce:
1. Load "fractal summer colors" theme details page on your Android device - https://addons-dev.allizom.org/en-US/android/addon/fractal-summer-colors/
2. Change the language to` Italiano` and observe add-on "name+author" section - https://addons-dev.allizom.org/it/android/addon/fractal-summer-colors/

Expected results:
Add-on name and author is displayed.

Actual results:
A number is displayed after theme author.

Notes/Issues:
- I was not able to reproduce this for other language or other theme. :(
- also, not being able to understand what that number means
- fact: searching for that number returns "fractal summer colors" theme

Verified on AMO-dev FF52(Android 6.0.1). reproducing all around AMO-servers.
Screenshots for this issue:
![2017-03-27_1633](https://cloud.githubusercontent.com/assets/15685960/24359220/5afff908-130c-11e7-937d-d3bc9401cf20.png)

**Comment by tofumatt** [comment]
Going to guess something snuck into the localisation text; I'll have a look.

**Comment by ValentinaPC** [comment]
Thanks!

**Comment by tofumatt** [comment]
Really odd that refreshing the page sometimes seems to not load it. 😮

Just checked and that data is coming from the API:

<img width="1316" alt="screenshot 2017-03-27 16 36 21" src="https://cloud.githubusercontent.com/assets/90871/24364728/9404036c-130b-11e7-8f2c-ca62f4ef378e.png">

So this seems to be an API bug because sometimes `addon.summary` is set as `null` and other times set as `'5335374'`.

Super weird. @diox, low priority as I'm guessing it's -dev only, but any clue what this could be?

**Comment by kumar303** [comment]
If `addon.summary` is sometimes a number this sounds like the number could be the translation database ID. There is some magic around swapping a translation ID with the actual translation string.

**Comment by eviljeff** [comment]
Personas don't have a `.summary` do they?  In devhub we only expose `.description`

**Comment by tofumatt** [comment]
In that case the API should returning null then. We check for null/empty summary and then display nothing… what’s odd is the API is returning that number. (See my screenshot)

**Comment by diox** [comment]
Yeah, it seems to be returning that number (which is definitely a translation id) only in search...

Here is where it gets weirded, in the db:
```
+---------+---------+--------+------------------+------------------------+---------------------+---------------------+
| autoid  | id      | locale | localized_string | localized_string_clean | created             | modified            |
+---------+---------+--------+------------------+------------------------+---------------------+---------------------+
| 5791582 | 5335374 | it     | 5335374          | 5335374                | 2013-04-12 08:55:51 | 2013-05-29 02:34:41 |
+---------+---------+--------+------------------+------------------------+---------------------+---------------------+
```

Yes, "5335374" is the actual text as well as the id... Seeing this, I'm surprised the detail API does not return it as well. I wonder if it's a problem specific to only this add-on - which seems to be an old-style Persona.

**Comment by diox** [comment]
```
[addons_dev_allizom_org]> SELECT COUNT(*) FROM translations WHERE CAST(id as CHAR(10)) = localized_string;        
+----------+
| COUNT(*) |
+----------+
|      452 |
+----------+
```

So we have 452 of those weird translations on dev - that's not a lot, but it's kinda annoying to see what they are attached to without going through all the possible fields on all possible models. Focusing on all Personas with summaries:

```
>>> Persona.objects.count()
384636
>>> Persona.objects.exclude(persona_id=0).count()
350804
>>> Persona.objects.exclude(persona_id=0).filter(addon__summary__isnull=False).count()    
855
>>> Persona.objects.filter(addon__summary__isnull=False).count()                      
2674
```

So, a bit more. Not a lot though, but more importantly some new-style Personas as well. At this point I'm not sure it's worth digging into this more - we should/could easily write a processing task to find them and reset their summaries to solve this issue.

**Comment by tofumatt** [comment]
👍🏻
Makes sense to me, no sense getting too deep in that rabbit hole, especially if it's just -dev. It being unknown is a bit worrying but then I think data on -dev in general can be a bit odd...

**Comment by ValentinaPC** [comment]
> Makes sense to me, no sense getting too deep in that rabbit hole, especially if it's just -dev

> reproducing all around AMO-servers.

**Comment by muffinresearch** [comment]
As @ValentinaPC points out, it's not just -dev so it would be good to get an idea of numbers elsewhere and work out the best solution in terms of clean-up.

**Comment by tofumatt** [comment]
Oh shoot, okay. My bad, sorry!

Matthew Riley MacPherson (sent from mobile)

**Comment by ValentinaPC** [comment]
> EDIT: Oh, I think @ValentinaPC is saying it happens on all instances of AMO. In that case this is extra weird.

At some point you saw that is happening all around :)

**Comment by diox** [comment]
I could not ssh to stage (again...) and I don't have prod access to gather the numbers there ; but the solution would be the same. There is no point in keeping the buggy summary references.

**Comment by diox** [comment]
This is purely a server problem but I'm too lazy to re-file it and add the history on addons-server, I'll have a PR that provides a command to remove the buggy summaries up shortly.

**Comment by ValentinaPC** [comment]
Verified as fixed on AMO-dev FF52(Android 6.0.1)
Verified for different languages and different personas.
Postfix screenshot: 
![2017-04-04_1505](https://cloud.githubusercontent.com/assets/15685960/24655754/34ee8558-1948-11e7-88ce-411227cdc53c.png)
